syntax = "proto3";
package Webservice;

message Start
{
	message ServiceInfo
	{
		string serviceName = 1;
		repeated string interestedColumnList = 2;
		string responseModelName = 3;	// model name of response
	}
	string host = 1;		// as the cert name and key name when apply SSL (https)
	string port = 2;
	repeated ServiceInfo serviceList = 3;
}

message PerformResponse
{
}
